package.json

```diff
@@ -24,7 +24,7 @@
   "devDependencies": {
     "@babel/core": "^7.23.9",
     "@layerzerolabs/eslint-config-next": "~2.3.3",
-    "@layerzerolabs/lz-definitions": "~2.3.3",
+    "@layerzerolabs/lz-definitions": "~2.3.24",
```

Comment:
Why did we need to deprecate this?

Reply:
this is an upgrade not deprecate, from 3 to 24. We need that because Etherlink endpoint ID v2 is not present in this old version.